docs/en/reference/basic-mapping.rst

```diff
@@ -50,30 +44,43 @@ that describes your entity.
 Doctrine provides several different ways to specify object-relational
 mapping metadata:
 
--  :doc:`Docblock Annotations <annotations-reference>`
 -  :doc:`Attributes <attributes-reference>`
+-  :doc:`Docblock Annotations <annotations-reference>`
 -  :doc:`XML <xml-mapping>`
--  :doc:`YAML <yaml-mapping>`
 -  :doc:`PHP code <php-mapping>`
+-  :doc:`YAML <yaml-mapping>` (deprecated and will be removed in ``doctrine/orm`` 3.0.)
 
 This manual will usually show mapping metadata via docblock annotations, though
 many examples also show the equivalent configuration in YAML and XML.
 
 .. note::
 
     All metadata drivers perform equally. Once the metadata of a class has been
-    read from the source (annotations, xml or yaml) it is stored in an instance
-    of the ``Doctrine\ORM\Mapping\ClassMetadata`` class and these instances are
+    read from the source (attributes, annotations, XML, etc.) it is stored in an instance
+    of the ``Doctrine\ORM\Mapping\ClassMetadata`` class which are
     stored in the metadata cache.  If you're not using a metadata cache (not
     recommended!) then the XML driver is the fastest.
 
 Marking our ``Message`` class as an entity for Doctrine is straightforward:
 
 .. configuration-block::
 
-    .. code-block:: php
+    .. code-block:: attribute
```

Comment:
As I commented on your commit where you mentioned me, isn't this going to break syntactic coloration?

Reply:
No, it's the same as I was using at https://www.doctrine-project.org/projects/doctrine-orm/en/2.11/reference/events.html#lifecycle-callbacks

Sorry about the mentioning! Somebody already told me that before - I hope I'll remember next time...

Reply:
No worries, please reword that commit to make it go away.


How to do that?

1. `git rebase -i origin/2.11.x`, assuming `origin` is a git remote that points to this repository, and not your fork. If you're not sure what your remotes are, run `git remote -vvv`, there should be your fork and the holy/reference/base/origin/whatever-you-call-it repository.
2. A window will show up with many lines, replace `pick` with `reword` on the line corresponding to your commit
3. Close your editor, git should do its magic, and let you amend the commit message
4. Use `git push --force` to overwrite what you already push. Don't forget the `--force` option otherwise git will try to merge both things together.

Reply:
Thanks, my "Git instructor"! :-) It looks like I succeeded.

Reply:
Indeed, great job! My inbox thanks you!

Reply:
So this is resolved? Could you please merge it then?